src/components/FeatureViews/Registration/RegistrationCourseEnrollments.js

```diff
@@ -86,8 +113,38 @@ const RegistrationCourseEnrollments = ({courseID}) => {
         }
     }
 
+    const handleClick = event => {
+        setStudentMenuAnchorEl(event.currentTarget);
+    };
+    const handleClose = event => {
+        setStudentMenuAnchorEl(null);
+    };
+
+    const handleUnenroll = (studentID, courseID, enrollment) => event => {
```

Comment:
No need for courseID here if it is already passed as a prop. Also, extract `studentID` from `enrollment` if you can